src/styles/blocks/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500px;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+}
```

Comment:
To make the title look more like the one on the layout, you can add font-style: normal.